src/Workspaces/Core/Portable/Classification/AbstractClassificationService.cs

```diff
@@ -146,6 +146,13 @@ private static bool IsFullyLoaded(Document document, CancellationToken cancellat
             ArrayBuilder<ClassifiedSpan> result,
             CancellationToken cancellationToken)
         {
+            if (options.ForceFrozenPartialSemanticsForCrossProcessOperations)
```

Comment:
nit: it feels to me that this option shouldn't really mention 'cross process'.  it's really: do i want frozen or not?

Reply:
For the single-process case, the argument itself determines whether frozen partial semantics are used (and in particular, the code changed by this PR affects that argument).